lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -25,6 +25,8 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args;
 
 $days_of_week = Get-AnsibleParam $params -name "days_of_week"
+$interval = Get-Attr $params -name "interval" -default $null
+$interval_unit = Get-Attr $params -name "interval_unit" -default minutes
 $enabled = Get-AnsibleParam $params -name "enabled" -default $true
 $enabled = $enabled | ConvertTo-Bool
 $description = Get-AnsibleParam $params -name "description" -default " "
```

Comment:
So nowadays we follow this syntax:

```powershell
$enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
$description = Get-AnsibleParam -obj $params -name "description" -type "str" -default " "
...
```

Possible types currently are:
- str
- path
- bool
- int

Also, I wonder why description defaults to " ", that seems a bit weird.